### ISSUE TITLE
Chunk upload process fails when AutoUpload and Multiple are set to false

### ISSUE THREAD

**Issue by veselints** [telerik/kendo-ui-core#3727]
### Bug report
The issue occurs in Upload widget, configured to ChunkUpload, AutoUpload = false and Multiple = false.

### Reproduction of the problem
* Go to the following [Dojo](https://dojo.telerik.com/oharEkIF);
* Select large file (20 MB) and click "Upload";
* While the file is still being uploaded select another file - this will stop the upload of the first file (expected), but it will also prevent the second from being available for upload;

The following error is thrown:
Uncaught TypeError: Cannot read property 'chunkIndex' of undefined

**KendoUI:** 2019.3.1023

**Comment by veselints** [comment]
The following *select* event handler implementation could be used as a workaround:

````JavaScript
select: function(ev) {
  var files = ev.sender.getFiles();
  for (var i = 0; i < files.length; i++) {
    ev.sender.clearFileByUid(files[i].uid);
  }
},
````

**Comment by jheitz1117** [comment]
This issue also happens when AutoUpload = false and Multiple=true and while uploading multiple chunked files concurrently. The suggested workaround does not work in this case. Is there something else I can try?

**Comment by rkaufman** [comment]
I ran into this issue as well and have found an answer that works. I am using the MVC framework so your back end may be different.
I found 3 key factors for this control. 1) The content is multipart/form-data so process accordingly. 2) The classes shown on the controller at this link: [https://demos.telerik.com/aspnet-mvc/upload/chunkupload](https://demos.telerik.com/aspnet-mvc/upload/chunkupload) are indeed the classes that are received and expected back. 3) The Javascript will fail if the returned object does not contain a property named "fileUid" and the Json serializer should use a CamelCasePropertyNamesContractResolver. Hope this helps in some way.
